routers/user/kitspace_auth.go

```diff
@@ -50,36 +69,41 @@ func KitspaceSignUp(ctx *context.Context, form auth.RegisterForm) {
 		}
 		return
 	} else {
-		// TODO: once email configuration is setup, the whole else block is no longer needed.
 		log.Trace("Account created: %s", u.Name)
 		response := make(map[string]bool)
 		response["IsRegisteredSuccessfully"] = true
 
-		ctx.JSON(http.StatusOK, response)
+		ctx.JSON(http.StatusCreated, response)
 		return
 	}
-
-	// Send confirmation email
-	// TODO: this needs updating the conf file in /custom
-	//if setting.Service.RegisterEmailConfirm && u.ID > 1 {
-	//	mailer.SendActivateAccountMail(ctx.Locale, u)
-	//
-	//	response := make(map[string]string)
-	//	response["IsRegisterMailSent"] = "true"
-	//	response["email"] = u.Email
-	//	response["ActiveCodeLives"] = timeutil.MinutesToFriendly(setting.Service.ActiveCodeLives,
-	//		ctx.Locale.Language())
-	//
-	//	if err := ctx.Cache.Put("MailResendLimit_"+u.LowerName, u.LowerName, 180); err != nil {
-	//		log.Error("Set cache(MailResendLimit) fail: %v", err)
-	//	}
-	//
-	//	ctx.JSON(http.StatusOK, response)
-	//	return
-	//}
```

Comment:
Good morning, I think we should still send the confirmation email? I want to test this on staging.kitspace.org later.  I thought you would un-comment this block...

Reply:
Good morning, I re-enabled it!